platform/core/src/services/UICustomizationService/CustomizationType.ts

```diff
@@ -0,0 +1,4 @@
+export default interface CustomizationTypeInfo extends Record<string, unknown> {
```

Comment:
Again we should stick to either Customization and CustomizationType OR UICustomization and UICustomizationType. It should not be UICustomizationService + CustomizationType + CustomizationModule

Reply:
Agreed, this is ugly.  I've deleted customizationType, as that one wasn't being used.  Will fix the others.